### PR TITLE
I5 update docs

### PR DESCRIPTION
**Pull request by labradford**
Added Code of Conduct
Updated MIT license to Apache license
Add Contributing document
Updated Readme with descriptions of features
* all links point to Samvera Labs url so we don't need to update later

References issue #23

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Contributing to Samvera::Persona
```
Comment (orangewolf) [inline CONTRIBUTING.md]: copy-pasta

File: README.md [hunk 2]
```diff
@@ -28,8 +26,43 @@ Install the migrations
 rails bulkrax:install:migrations db:migrate
```
Comment (orangewolf) [inline README.md]: take this install migrations step out as its not actually needed. Just have them run db:migrate.

File: README.md [hunk 3]
```diff
@@ -28,8 +26,43 @@ Install the migrations
...
 ```
 
+
+## Requirements
```
Comment (orangewolf) [inline README.md]: remove blank section?

File: README.md [hunk 4]
```diff
@@ -1,8 +1,6 @@
 # Bulkrax
-Short description and motivation.
+Bulkrax is an import gem that provides an admin interface to create and run importers for your Hyrax applications.
```
Comment (orangewolf) [inline README.md]: What do you think about punching this up a little? Something like 

> Bulkrax is a batteries included importer for Samvera applications. It currently includes support for OAIPMH (DC and Qualified DC) and CSV out of the box. It is also designed to be extensible, allowing you to easily add new importers in to your application or to include them with other gems. Bulkrax provides a full admin interface including creating, editing, scheduling and reviewing imports.

File: CONTRIBUTING.md (hunk 1 above)
Comment (labradford) [reply CONTRIBUTING.md]: fixed typo

File: README.md (hunk 2 above)
Comment (labradford) [reply README.md]: removed migration step

File: README.md (hunk 3 above)
Comment (labradford) [reply README.md]: removed

File: README.md (hunk 4 above)
Comment (labradford) [reply README.md]: I think this is much better than what I had.

**Pull request merged**